src/serializers/OrganizationSerializer.ts

```diff
@@ -21,7 +21,7 @@ import { Serializer, SerializerOptions } from 'jsonapi-serializer';
 
 import { PaginationLinks } from '.';
 
-export const ORG_ATTRIBUTES: string[] = ['id', 'slug', 'name', 'owners'];
+export const ORG_ATTRIBUTES: string[] = ['id', 'slug', 'name', 'owners', 'errors'];
```

Comment:
We don't need to add extra properties here, since errors are handled by the above createUserBulkSerializer serializer.